tests/ha/test_server_pod_failure.py

```diff
@@ -1323,84 +1217,108 @@ def test_reads_writes_during_server_pod_down(self):
         LOGGER.info("STARTED: Test to verify READs/WRITEs during server pod down by "
                     "delete deployment.")
         event = threading.Event()  # Event to be used to send intimation of server pod deletion
-        LOGGER.info("Step 1: Perform READs/WRITEs with variable object sizes during "
-                    "server pod down by delete deployment.")
         users = self.mgnt_ops.create_account_users(nusers=1)
-        self.test_prefix = 'test-39911'
         self.s3_clean = users
-        output = Queue()
 
-        args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
-                'nclients': 1, 'nsamples': 10, 'skipcleanup': True, 'output': output}
-
-        thread = threading.Thread(target=self.ha_obj.event_s3_operation,
-                                  args=(event,), kwargs=args)
-        thread.daemon = True  # Daemonize thread
-        thread.start()
+        LOGGER.info("Step 1: Perform READs/WRITEs with variable object sizes during "
+                    "server pod down by delete deployment.")
+        LOGGER.info("Step 1.1: Perform WRITEs with variable object sizes for parallel READs")
+        test_prefix_read = 'test-read-39911'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=test_prefix_read, skipread=True,
+                                                    skipcleanup=True, nclients=5, nsamples=5)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1.1: Performed WRITEs with variable sizes objects for parallel READs.")
+
+        LOGGER.info("Step 1.2: Start WRITEs with variable object sizes in background")
+        test_prefix_write = 'test-write-39911'
+        output_wr = Queue()
+        args = {'s3userinfo': list(users.values())[0], 'log_prefix': test_prefix_write,
+                'nclients': 1, 'nsamples': 5, 'skipread': True, 'skipcleanup': True,
```

Comment:
Please increase nsamples. 5 samples will be over within 30 seconds.

Reply:
We can update the same when TE task. This was already once tested.